test/shared/extends.js

```diff
@@ -0,0 +1,29 @@
+var cons = require('../../')
+  , fs = require('fs');
+
+var should = require('should');
+
+exports.test = function(name) {
+  var user = { name: 'Tobi' };
+
+  describe(name, function(){
+
+    if (name == 'nunjucks') {
```

Comment:
I like it better to keep this as another possibility in the original test. Otherwise this is quite some overhead and it'll also print a empty description for other libraries.